src/components/common/checkbox/index.tsx

```diff
@@ -0,0 +1,71 @@
+import { useEffect, useState } from "react";
+
+import { Highlight } from "@/components/common/highlight";
+import { useAppDispatch, useAppSelector } from "@/redux/hooks";
+import { setBrand } from "@/redux/slices/filter-params-slice";
+
+import type { CheckboxProps } from "./checkbox.type";
+
+const Checkbox: React.FC<CheckboxProps> = ({
```

Comment:
```suggestion
const BranchCheckbox: React.FC<CheckboxProps> = ({
```
Components này e đang set cứng xử lý cho branches rồi, k dùng lại được chỗ khác nên có thể đặt tên cụ thể hơn

Reply:
done